sidecar/sokuji_sidecar/hf_symlinks.py

```
@@ -0,0 +1,82 @@
+        # previously shared the exact-named `p + ".tmp"`, so one could
+        # remove/replace the other's in-progress tmp between creation and
+        # os.replace, raising FileNotFoundError. A stale tmp left behind by a
+        # run that crashed before reaching os.replace (a different, now-dead
+        # pid) is swept below, best-effort, before this process starts its own.
+        tmp = f"{p}.tmp{os.getpid()}"
+        for stale in glob.glob(f"{p}.tmp*"):
+            try:
+                os.remove(stale)
+            except OSError:
+                try:
+                    os.chmod(stale, 0o666)  # Windows: a read-only blob copy can't be unlinked as-is
```

Ruff (0.15.21)

[error] `os.chmod` setting a permissive mask `0o666` on file or directory

(S103)